Vicare dhw schedule service

CrEaK opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
I have added a service to configure the schedule for my circulation pump. This allows disabling the pump when nobody is at home and re-enabling it when someone returns.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/137856

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
  - Introduced a new option that lets users set a schedule for domestic hot water pump circulation.

- **Bug Fixes**
  - Refined descriptive text for mode settings to improve clarity.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11801 to 0.5.11805 (#37322)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11801 to 0.5.11805.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump parser from 3.3.7.0 to 3.3.7.1 (#37321)

Bumps [parser](https://github.com/whitequark/parser) from 3.3.7.0 to 3.3.7.1.
- [Changelog](https://github.com/whitequark/parser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/whitequark/parser/compare/v3.3.7.0...v3.3.7.1)

---
updated-dependencies:
- dependency-name: parser
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

c0ffeeca7 committed:
When downloading from backup page, backup is decrypted (#37324)

* When downloading from backup page, backup is decrypted

* Update source/_includes/common-tasks/backups.md

Co-authored-by: Franck Nijhof <git@frenck.dev>

---------

Co-authored-by: Franck Nijhof <git@frenck.dev>

starkillerOG committed:
Add new Reolink push callback info (#37325)

c0ffeeca7 committed:
Storing emergency kit: clarify steps (#37326)

* Storing emergency kit: clarify steps

* Apply parallelism

ZephireNZ committed:
Add Bluesky and Mastodon links, and add redirects (#37329)

karwosts committed:
Explain use of `collection_key` for energy cards (#37335)

* Update energy.markdown

* Update energy.markdown

* Typo

---------

Co-authored-by: Franck Nijhof <git@frenck.dev>

thecode committed:
Update Shelly Bluetooth device requirements (#37330)

karwosts committed:
Update statistics-graph.markdown (#37337)

smhc committed:
Add IRK instructions for android (#37117)

* Add IRK instructions for android

* Update source/_integrations/private_ble_device.markdown

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

* Tiny tweaks

---------

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

emufan committed:
Update openai_conversation.markdown (#37086)

TheCodeJanitor-dotcom committed:
Update squeezebox.markdown (#37293)

* Update squeezebox.markdown

Adds specific examples for entering call_method in the Visual Editor with working parameter syntax, as well as showing the properly formed YAML generated.

Squeezebox increment / decrement parameters require a preceding plus or minus, which will be dropped by the Visual Editor if not double-quoted.

* Update squeezebox.markdown

Correct missing closing pipes in tables per Code Rabbit suggestion.

Zemoj committed:
Update troubleshooting.markdown (#37265)

* Update troubleshooting.markdown

Added media source fix for no TTS in YAML config.

* Update troubleshooting.markdown

streamlined media source text and added refence for default config. added local network header for more consistent style.

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.23.8 to 0.23.9 (#37341)

Bumps [ruby-lsp](https://github.com/Shopify/ruby-lsp) from 0.23.8 to 0.23.9.
- [Release notes](https://github.com/Shopify/ruby-lsp/releases)
- [Commits](https://github.com/Shopify/ruby-lsp/compare/v0.23.8...v0.23.9)

---
updated-dependencies:
- dependency-name: ruby-lsp
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

davet2001 committed:
Fix weather map example to be one that works (#37339)

lellky committed:
Clarify hardware needs for flexit_bacnet (#37348)

* Clarify hardware needs for flexit_bacnet

* Fix typo

* Tiny tweaks

---------

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck committed:
2025.2.1 (#37353)

davet2001 commented:
Hi @CrEaK thank you for the PR!

There seem to be unrelated changes, please can you rebase or otherwise reduce the changes to be just related to the PR?

Thanks!

niklasneesen committed:
added dokumentation for new service set_dhw_circulation_pump_schedule

CrEaK commented:
@davet2001 Fixed the history now. Sorry for that!

davet2001 commented on source/_integrations/vicare.markdown:
```diff
@@ -107,7 +107,7 @@ Number entities are available to adjust values like the predefined temperature f
 The following actions of the [climate integration](/integrations/climate/) are provided by the ViCare integration: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
 
-The following actions of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature`
+The following actions of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature', ``
```
There seems to be a formatting error here

davet2001 commented on source/_integrations/vicare.markdown:
```diff
@@ -118,6 +118,15 @@ Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode]
+
+| Data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```
```suggestion
| `entity_id` | yes | String or list of strings that point at `entity_id`s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```

davet2001 commented on source/_integrations/vicare.markdown:
```diff
@@ -118,6 +118,15 @@ Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode]
+| Data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`'s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
+| `schedule` | no | New schedule as json-string. Example of json-schema: https://www.postman.com/vimicho/viessmann-api-public/request/hpbagam/set-heating-schedule |
```
```suggestion
| `schedule` | no | New schedule as json-string. Example of json-schema: https://www.postman.com/vimicho/viessmann-api-public/request/hpbagam/set-heating-schedule |
```
Looks ok but I don't know anything about the linked API test page(?), or whether it's acceptable to link to 3rd party examples in the docs.

Is it possible to paste the example json into this page?

CrEaK replied:
Yeah sure I just use the json example from there.

niklasneesen committed:
small changes

davet2001 commented on source/_integrations/vicare.markdown:
```diff
@@ -107,17 +107,26 @@ Number entities are available to adjust values like the predefined temperature f
 The following actions of the [climate integration](/integrations/climate/) are provided by the ViCare integration: `set_temperature`, `set_hvac_mode`, `set_preset_mode` 
 
-The following actions of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature`
+The following actions of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature'
```
```suggestion
The following actions of the [water_heater integration](/integrations/water_heater/) are provided by the ViCare integration: `set_temperature`
```

davet2001 reviewed (changes requested):
Minor correction to formatting needed. Otherwise looks ok although I have not checked the API content.

jsuar committed:
Add info about multiple targets support for Slack integration (#37371)

* Add info about multiple targets

* Blank line before the list

---------

Co-authored-by: John Suarez <suarezjg@gmail.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11805 to 0.5.11813 (#37389)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11805 to 0.5.11813.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

calebmah committed:
Add documentation for message_thread_id data attribute for telegram notify integration (#37381)

* Add documentation and examples for message_thread_id data attribute

* Remove trailing colon in headers

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tr4nt0r committed:
Improve Bring! integration documentation (#37340)

* Improve Bring! integration documentation

* bad gateway, custom polling

* add social login howto

* tiny style tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

ZephireNZ committed:
Improve `flick_electric` documentation (#37388)

* Improve flick_electric documentation

* Fix grammar

* Remove scale for now

* tiny tweak

* remove redundant fields in configuration_basic block

* add abbreviation tooltip

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tr4nt0r committed:
Update notification texts in Bring (#37395)

* Update notification texts in Bring

* tiny tweak

* revert tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
badge visibility: add link to available conditions (#37396)

EvertJob committed:
Update season.markdown (#37398)

mrdarrengriffin committed:
Remove unused country code selection (#37397)

EvertJob committed:
Update season.markdown (#37403)

Removed the round brackets

frenck committed:
2025.2.2 (#37404)

dependabot[bot] committed:
build(deps-dev): bump json from 2.9.1 to 2.10.1 (#37412)

dependabot[bot] committed:
build(deps-dev): bump ruby-lsp from 0.23.9 to 0.23.10 (#37411)

Bre77 committed:
Update tesla_fleet.markdown (#37410)

G1z committed:
Example correction (#37406)

tjb36 committed:
Update shopping_list.markdown (#37405)

Added documentation of the "remove" action which is present in the shopping_list_updated event that is fired when an item is removed from the list. This was not present before (only add and update were).

c0ffeeca7 committed:
motionEye: update tap action keywords in example (#37413)

- to reflect current software
- using keywords as described in https://www.home-assistant.io/dashboards/actions/#action

c0ffeeca7 committed:
Rpi imager: move link to image download into procedure (#37419)

CrEaK committed:
Update source/_integrations/vicare.markdown

Co-authored-by: Dave T <17680170+davet2001@users.noreply.github.com>

niklasneesen committed:
small changes to formatting

niklasneesen committed:
added dokumentation for new service set_dhw_circulation_pump_schedule

abmantis commented on source/_integrations/vicare.markdown:
```diff
@@ -115,9 +115,32 @@ Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode]
+
+| Data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```
since it is optional, we should also describe what happens when it is not provided

CrEaK replied:
in reality its not optional at all. changed it to no :-)

abmantis replied:
still unresolved

abmantis commented on source/_integrations/vicare.markdown:
```diff
@@ -115,9 +115,32 @@ Set the mode for the climate device as defined by Viessmann (see [set_hvac_mode]
+
+| Data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of strings that point at `entity_id`s of climate devices to control. To target all entities, use the `all` keyword instead of entity_id. |
```
```suggestion
| `entity_id` | yes | `entity_id` or list of `entity_id`s of climate devices to control. To target all entities, use the `all` keyword instead of an entity_id. |
```
IMO, the descriptions of the other services that have this wording should also be updated (but not in this PR), as they are unnecessarily complex.

abmantis replied:
still unresolved

abmantis reviewed (changes requested):
Something went wrong with the latest updates because a lot of files show up on the diff now.

CrEaK commented:
Fixed now sorry

abmantis reviewed (changes requested):
It seems that your latest changes are now missing
